components/SegmentedControl/src/EqualPanel.cs

```diff
@@ -66,25 +74,6 @@ protected override Size ArrangeOverride(Size finalSize)
         }
         return finalSize;
     }
```

Comment:
@niels9001 Note: for your other issue, you're ignoring `finalSize` in the arrange method, so that's why it's not working in the other scenario.